sites/docs/src/content/docs/contributing/nf-test/assertions.md

```diff
@@ -97,12 +103,13 @@ log[0][1]
 # nf-core guidelines for assertions
 
 1. **Encapsulate Assertions in `assertAll()`**: Group all assertions within `assertAll()` for comprehensive testing.
+
 2. **Minimum Requirement - Process Success + version.yml file**: Always check if the process completes successfully and make at least a snapshot of the version.yml.
 
 ```groovy
 assertAll(
     { assert process.success },
-    { assert snapshot(process.out.versions).match("versions") }
+    { assert snapshot(process.out.versions).match() }
```

Comment:
We should probably swap to using the explicit versions yml option (wherever the link to that is)

Reply:
feel free to update and merge :)

Reply:
I'd need to find the explicit versions yml code first ;)